### ISSUE TITLE
SmartCity-Accessibility #101 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#96]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 49 lines...]

[WARNING] The POM for teamdev:license:jar:1.0 is missing, no dependency information available
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/lang-tag/maven-metadata.xml


Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml


[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/workspace/SmartCity-Accessibility/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 49 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/AuthenticatedUser.java:[10,8] smartcity.accessibility.socialnetwork.AuthenticatedUser is not abstract and does not override abstract method setFavouriteSearchQueries(java.util.List<smartcity.accessibility.search.SearchQuery>) in smartcity.accessibility.socialnetwork.User
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 15.580 s
[INFO] Finished at: 2016-12-27T00:46:46+00:00
[INFO] Final Memory: 26M/170M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/AuthenticatedUser.java:[10,8] smartcity.accessibility.socialnetwork.AuthenticatedUser is not abstract and does not override abstract method setFavouriteSearchQueries(java.util.List<smartcity.accessibility.search.SearchQuery>) in smartcity.accessibility.socialnetwork.User
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Arthur Sapozhnikov] 350b225e858e55e619017354bc867a96688712b3 - Added basic implementation for the new User design, issue #95, please

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/101/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 49 lines...]

[WARNING] The POM for teamdev:license:jar:1.0 is missing, no dependency information available
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/lang-tag/maven-metadata.xml


Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml


[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] Deleting /var/lib/jenkins/workspace/SmartCity-Accessibility/target
[INFO] 
[INFO] --- jacoco-maven-plugin:0.7.8:prepare-agent (default-cli) @ SmartCity-Accessibility ---
[INFO] argLine set to -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/workspace/SmartCity-Accessibility/target/jacoco.exec
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 49 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/AuthenticatedUser.java:[10,8] smartcity.accessibility.socialnetwork.AuthenticatedUser is not abstract and does not override abstract method setFavouriteSearchQueries(java.util.List<smartcity.accessibility.search.SearchQuery>) in smartcity.accessibility.socialnetwork.User
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 14.859 s
[INFO] Finished at: 2016-12-27T00:53:55+00:00
[INFO] Final Memory: 26M/168M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/socialnetwork/AuthenticatedUser.java:[10,8] smartcity.accessibility.socialnetwork.AuthenticatedUser is not abstract and does not override abstract method setFavouriteSearchQueries(java.util.List<smartcity.accessibility.search.SearchQuery>) in smartcity.accessibility.socialnetwork.User
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Arthur Sapozhnikov] 350b225e858e55e619017354bc867a96688712b3 - Added basic implementation for the new User design, issue #95, please

- [Ariel Kolikant] 29d1f6200009ab9f88af37be59ca34c0a897c481 - git says i changed something, no idea what though



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/102/)

**Comment by kapalex** [comment]
Just for ther record @Kolikant , the issue is closen automatically when the build is fixed, no need to close it manually
